backend/maelstro/metadata/meta.py

```diff
@@ -79,7 +79,35 @@ def get_gslayer_from_gn_link(
             workspace_name=ows_url.lstrip("/").split("/")[0], layer_name=layer_name
         )
 
-    def update_geoverver_urls(self, mapping: dict[str, list[str]]) -> tuple[str, str]:
+    def _apply_xslt(self, xslt_path: str) -> bytes:
+        with open(xslt_path, encoding="utf8") as xslt_file:
+            xslt = etree.parse(xslt_file)
+        transform = etree.XSLT(xslt)
+        xml_root = etree.parse(BytesIO(self.xml_bytes))
+        return self.dump_tree_to_bytes(transform(xml_root))
+
+    def apply_xslt(self, xslt_path: str) -> tuple[str, str]:
+        pre = len(self.xml_bytes)
+        self.xml_bytes = self._apply_xslt(xslt_path)
+        post = len(self.xml_bytes)
+        return f"Before: {pre} bytes", f"After: {post} bytes"
+
+    def apply_xslt_chain(self, xslt_paths: list[str]) -> tuple[str, str]:
```

Comment:
Same, return a string, and same in MetaZip methods

Reply:
no, it is called by the child class MetaZip via super()
https://github.com/georchestra/maelstro/blob/GSREN-237_xsl_transform/backend/maelstro/metadata/meta.py#L191